docs/usage/configuration-options.md

```diff
@@ -2109,7 +2109,28 @@ In the case that a user is automatically added as reviewer (such as Renovate App
 
 ## ignoreScripts
 
-Applicable for npm, bun, Composer and Copier only for now. Set this to `true` if running scripts causes problems.
+By default, Renovate will not run any scripts.
+
+### No script execution on free Mend-hosted Renovate
+
+The Mend Renovate App does not allow scripts to run.
+We do not plan to let users on free tiers run scripts, because the risk of abuse is too high.
+
+### Renovate Enterprise Cloud can be configured to run scripts
+
+Scripts can be enabled for paying customers on Mend.io hosted apps.
+Please ask Mend.io sales about "Renovate Enterprise Cloud".
+
+### Allowing scripts if self-hosting Renovate
+
+If you are self-hosting Renovate, and want Renovate to run scripts:
+
+1. Set the self-hosted config option [`allowScripts`](../self-hosted-configuration.md#allowscripts) to `true` in your bot/admin configuration
+1. Set `ignoreScripts` to `false` for the package managers you want to allow to run scripts (only works for the listed supportedManagers in the table above)
+
+#### Example config to allow scripts for npm
+
+Insert example config that allows script execution for just the npm package manager.
```

Comment:
Can you configure Renovate like this?

1. Allow script execution, by setting the self-hosted config option `allowScripts=true`.
2. But limit the risk, by only allowing the `npm` package manager to run scripts.

Reply:
Actually, `allowScripts` is a top-level global setting so is turned on for all managers or none

Reply:
I dropped the section about the example config to allow scripts for just npm.